tracer/test/snapshots/WeakHashingTests.SubmitsTraces.verified.txt

```diff
@@ -0,0 +1,695 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: weak_hashing,
+    Resource: weak_hashing,
+    Service: Samples.WeakHashing-weak_hashing,
```

Comment:
If these spans appear inside normal APM traces, we probably want the service name to be `Samples.WeakHashing`, without the `-weak_hashing` suffix. The suffix should only be used for "client spans" that represent calls to another downstream service (e.g. http, sql, redis).

Reply:
Good to know. Thanks!